### DISCUSSION TITLE
[Feature Sugestion] Allow indexing of assets in subfolders

### DISCUSSION THREAD

**Discussion by Typhonragewind** [neonwatty/meme-search#19] [Ideas]
Pretty much what it says in the title. It would be useful to be able to use subfolders on the main data folder:

- Easier to import existing images
- Keeps some low tech organization in case of app technical issues
- In docker, allows for easy mounting of other folders without having to move pictures around

**Comment by neonwatty** [comment]
Solved in the new "pro" version of the app!
